plugins/onepad/keyboard.cpp

```diff
@@ -99,7 +99,7 @@ bool PollForNewKeyboardKeys(u32 &pkey) {
     // keys that aren't being recognized, bump this number up!
     for (int key = 0; key < 0x80; key++) {
         if (CGEventSourceKeyState(kCGEventSourceStateHIDSystemState, key)) {
-            pkey = key == kVK_Escape ? 0 : key;
+            pkey = key == kVK_Escape ? -1 : key;
```

Comment:
pKey is unsigned, and it's potentially being set to -1.

Reply:
Would you prefer `(u32)-1` or `0xFFFFFFFF`?

Reply:
Why not ```UINT32_MAX```